src/persistedState/codecs.ts

```diff
@@ -107,6 +107,18 @@ export const persistedStateCodecNew = buildTypeCodec({
 			selectedCaseHash: null,
 		},
 	),
+	panelView: withFallback(
+		buildTypeCodec({
+			kind: t.union([t.literal('JOB'), t.literal('CODEMOD'), t.null]),
+			docs: t.union([t.string, t.null]),
+			title: t.string,
```

Comment:
actually, for the `JOB`, the title will be calculated from other state properties.